sort/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace sort
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int i;
+            int[] a = new int[30];
```

Comment:
If you will move this down to below `int n = ...` you can create an array only as big as you need.